codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/ErrorsGeneratorBase.java

```diff
@@ -162,23 +163,29 @@ public void renderRbsErrorCode() {
     }
 
     private void renderServiceModelErrors() {
-        errorShapes.forEach(error -> {
-            String errorName = symbolProvider.toSymbol(error).getName();
-            ErrorTrait errorTrait = error.getTrait(ErrorTrait.class).get();
-            String apiErrorType = getApiErrorType(errorTrait);
-
-            renderServiceModelError(errorName, apiErrorType);
-        });
+        errorShapes
```

Comment:
Is this refactorable so we don't have to do it twice?

Reply:
Yes, but technically changes the interface slightly (set to list).  fine for pre-release here, so I've done that.